package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "validate-commit-msg",
   "description": "Script to validate a commit message follows the conventional changelog standard",
-  "main": "index.js",
+  "main": "dist/index.js",
   "version": "0.0.0-development",
   "scripts": {
+    "build:watch": "babel -w ./index.js -d dist",
+    "prebuild": "rimraf dist",
+    "build": "babel --out-dir dist ./index.js",
     "commit": "git-cz",
-    "check-coverage": "istanbul check-coverage --statements 100 --branches 90 --functions 100 --lines 100",
+    "check-coverage": "nyc check-coverage --statements 100 --branches 90 --functions 100 --lines 100 --compilers js:babel-register",
```

Comment:
I don't think the `--compilers js:babel-register` flag is necessary here.

And actually, as of v7 of nyc, you can now add coverage thresholds as part of the configuration and we wont even need this script at all! See [this](https://github.com/kentcdodds/starwars-names/blob/FEM/10.1-docs/package.json#L66-L70) for an example (note, if you want you can actually look at a few other differences this has with the lessons you watched because things have improved slightly since the videos were recorded).